### PR TITLE
EES-4888 - Update github libraries to latest versions

### PR DESCRIPTION
**Pull request by RTodorov**
Updating github libraries to address incompatibilities in the requests that are causing 500 errors on `hiring unseat` command.

### CODE REVIEW

**Comment by vgarvardt** [comment]
👍 

Approved with Zappr  Approved with Zappr  Approved with Zappr  Approved with Zappr

File: go.mod [hunk 1]
```diff
@@ -6,20 +6,20 @@ require (
 	github.com/dghubble/sling v1.1.0
 	github.com/dsnet/compress v0.0.0-20171208185109-cc9eb1d7ad76 // indirect
 	github.com/emirpasic/gods v1.12.0 // indirect
+	github.com/go-git/go-git/v5 v5.2.0
 	github.com/golang/protobuf v1.4.3 // indirect
 	github.com/golang/snappy v0.0.0-20170215233205-553a64147049 // indirect
-	github.com/google/go-github v13.0.1-0.20171014143926-a021c14a5f19+incompatible
-	github.com/google/go-querystring v0.0.0-20170111101155-53e6ce116135 // indirect
+	github.com/google/go-github v17.0.0+incompatible // indirect
```
Comment (RTodorov) [inline go.mod:12]: This is because of `italolelis/goupdater`

File: go.mod [hunk 2]
```diff
@@ -6,20 +6,20 @@ require (
...
+	github.com/google/go-github/v33 v33.0.0
 	github.com/hashicorp/errwrap v0.0.0-20141028054710-7554cd9344ce
 	github.com/hashicorp/go-multierror v0.0.0-20171204182908-b7773ae21874
 	github.com/hashicorp/hcl v0.0.0-20171017181929-23c074d0eceb // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/italolelis/goupdater v0.1.0
```
Comment (RTodorov) [inline go.mod:18]: we should probably replace this

**Comment by aleksandrzhiliaev** [comment]
+1

Approved with Zappr  Approved with Zappr  Approved with Zappr  Approved with Zappr

**Review by aleksandrzhiliaev — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (vgarvardt) [reply go.mod:18]: This one is good candidate - `github.com/hellofresh/updater-go` - we use it in couple other projects

**Pull request merged**